src/Components/test/testassets/BasicTestApp/BasicTestApp.csproj

```diff
@@ -19,4 +19,17 @@
     <ProjectReference Include="..\TestContentPackage\TestContentPackage.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Update="Resources.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>Resources.resx</DependentUpon>
+    </Compile>
+
+    <EmbeddedResource Update="Resources.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+  </ItemGroup>
```

Comment:
Just out of interest, is this stuff required? I'm surprised this isn't the default for `resx` within the SDK. Seems a bit inelegant (though not your fault, obviously).

Reply:
Yes, it doesn't generate the designer file without this.